.travis.yml

```diff
@@ -7,18 +7,16 @@ jobs:
     - stage: prechecks
       python: 3.6
       env: TOXENV=pylint
```

Comment:
Isn't `pylint` done with the `formatting` checks from pre-commit?

Reply:
You're right we can actually optimize that !